install/kubernetes/cilium/values.schema.json

```diff
@@ -5657,4 +5660,4 @@
   },
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object"
-}
\ No newline at end of file
+}
```

Comment:
nit: This file is auto-generated, thus this removal of the newline here shouldn't happen

Reply:
@gandro thanks for reviewing. You are right. Fixed it by running `make cilium/values.schema.json`